eegrasp/eegrasp.py

```diff
@@ -27,10 +31,15 @@ def __init__(self, data=None, coordinates=None, labels=None):
         """
         Parameters
         ----------
-        data: 2-d array, where the first dim are channels and the second is
-        samples.  
-        Coordenates: ndim array with position of the electrodes.
-        labels: 1-d array with the channel names.
+        data : ndarray | mne object | None.
```

Comment:
Specify which MNE object.

Reply:
Specified 3 mne objects. This covers the uses for averaged epochs or erps, epochs and continuous data which covers the analysis of resting state data.

Reply:
I don't see the change.

Reply:
Haven't made the commit yet